src/BudgetManager/BudgetManager.SDK/DTOs/AddCategoryDTO.cs

```diff
@@ -12,6 +12,10 @@ public class AddCategoryDTO
 
         public OperationType CategoryType { get; set; }
 
-        public string Color { get; set; }
+        public string Color
+        {
+            get => CategoryType == OperationType.Income ? "green.500" : "red.500";
+            set => Color = value;
+        }
```

Comment:
Is this correct? Because in fe team said, if value is not set, it should be null. Besides this would always return green or red, no matter what color is set, right?

Reply:
@Saigunas I wanted to assign an initial value for color based on the value of category type. So, it's not a correct implementation, right? Should do something like this:
` public string Color { get; set; } = null;`

Reply:
![изображение](https://user-images.githubusercontent.com/48493712/185604871-24148a82-0fab-4ecf-981d-c12ba47cad48.png)

Reply:
> @Saigunas I wanted to assign an initial value for color based on the value of category type. So, it's not a correct implementation, right? Should do something like this: ` public string Color { get; set; } = null;`

I think yes, if unset -> make it null, if set -> the value, that was set